frontend/i18n-scripts/export-pos.sh

```diff
@@ -2,22 +2,24 @@
 
 set -exuo pipefail
 
+source ./i18n-scripts/languages.sh
+
 for f in public/locales/en/* ; do
-  yarn i18n-to-po -f $(basename "$f" .json) -l zh-cn
-  yarn i18n-to-po -f $(basename "$f" .json) -l zh-tw
-  yarn i18n-to-po -f $(basename "$f" .json) -l ja
-  yarn i18n-to-po -f $(basename "$f" .json) -l ko
+  for i in "${LANGUAGES[@]}"
+  do
+  yarn i18n-to-po -f $(basename "$f" .json) -l $i
```

Comment:
It's a good practice to use quotes in case a filename could ever have special characters or spaces.

```suggestion
  yarn i18n-to-po -f "$(basename "$f" .json)" -l $i
```